src/rickbox/database/sql/procs/__init__.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
Naming here should probably be changed. The names imply these allow you to call the stored procs – that's coming after getting stored procs setup easily.

Ditto on "Maybe this should all be a shell script instead"